protocol/communities/manager.go

```diff
@@ -1221,7 +1221,15 @@ func (m *Manager) ReevaluateCommunityMembersPermissions(community *Community) er
 		return ErrNotEnoughPermissions
 	}
 
+	// Publish when the reevluation started since it can take a while
+	signal.SendCommunityMemberReevaluationStarted(community.IDString())
+
 	newPrivilegedMembers, err := m.ReevaluateMembers(community)
+
+	// Publish the reevaluation ending, even if it errored
+	// A possible improvement would be to pass the error here
+	signal.SendCommunityMemberReevaluationEnded(community.IDString())
+
```

Comment:
I thin it would be better to keep it inside `ReevaluateMembers` maybe? 🤔

Reply:
It's simpler to put it in `reevaluateCommunityMembersPermissions`, because it's the one that gets called by others and the only one that calls `ReevaluateMembers`.

Also, I want to call `SendCommunityMemberReevaluationEnded` whether it is a failure or not, and doing that in `ReevaluateMembers` would be a pain, because I'd have to put it before every `return`s